reporter-web-app/src/App.css

```diff
@@ -249,3 +254,8 @@ ul.ort-table-list {
1.4,32.4,488.2,38.6z'/%3e%3cpath class='st95' d='M553.2,38.6l-5.7,6.5c-6.4-5.1-12.2-7.3-19.3-7.3c-8.9,0-15.6,4.3-15.6,12.5c0,7.5,3.5,11,18.7,15.6 c13.8,4.2,24.7,9.5,24.7,25.3c0,14.7-11,24.8-29.4,24.8c-12,0-21.3-4-28.2-10.8l5.9-6.6c6.4,5.3,12.9,8.6,22.2,8.6 c10.3,0,18.6-5.3,18.6-15.6c0-8.7-4.2-12.5-18.6-16.9c-16.5-5-24.7-11.2-24.7-24.1c0-12.6,10.8-21.6,26-21.6 C539.2,28.9,546.4,32.4,553.2,38.6z'/%3e%3cpath class='st95' d='M381.1,193.6h-7.8V226h-20v-84.7H381c23,0,34.8,8.2,34.8,25.8c0,11.1-5.3,17.8-16.2,22.8l21.5,36h-22.6 L381.1,193.6z M373.3,179.8h8.4c8.8,0,13.4-3.8,13.4-12.7c0-8.2-4.8-11.7-14.7-11.7h-7.2V179.8z'/%3e%3cpath class='st95' d='M483.1,199.2h-40.2c1.3,11.6,6.8,14.8,14.8,14.8c5.3,0,9.9-1.8,15.5-5.7l7.9,10.8c-6.5,5.1-14.8,9-25.3,9 c-21.6,0-32.6-13.9-32.6-34.2c0-19.4,10.6-34.8,30.3-34.8c18.6,0,29.9,12.2,29.9,33.2C483.5,194.3,483.4,197.3,483.1,199.2z M464.5,186.8c-0.1-8.8-2.8-14.9-10.5-14.9c-6.5,0-10.3,4.2-11.1,15.8h21.6V186.8z'/%3e%3cpath class='st95' d='M529,226h-22.8l-20.9-64.9h21l11.5,50.2l12-50.2h19.7L529,226z'/%3e%3cpath class='st95' d='M577.2,137.9c0,6.2-4.8,11-11.5,11s-11.4-4.8-11.4-11c0-6.2,4.6-11,11.4-11S577.2,131.7,577.2,137.9z M575.5,226h-19.3v-64.9h19.3V226z'/%3e%3cpath class='st95' d='M646.7,199.2h-40.2c1.3,11.6,6.8,14.8,14.8,14.8c5.3,0,9.9-1.8,15.5-5.7l7.9,10.8c-6.5,5.1-14.8,9-25.3,9 c-21.6,0-32.6-13.9-32.6-34.2c0-19.4,10.6-34.8,30.3-34.8c18.6,0,29.9,12.2,29.9,33.2C647,194.3,646.9,197.3,646.7,199.2z M628.1,186.8c-0.1-8.8-2.8-14.9-10.5-14.9c-6.5,0-10.3,4.2-11.1,15.8h21.6V186.8z'/%3e%3cpath class='st95' d='M728,226h-23.1l-8.7-47.6l-8.6,47.6h-22.8l-13.8-64.9h19.5l6.7,50.5l10.1-50.5h18.8l8.9,50.5l7.8-50.5h18.4 L728,226z'/%3e%3cpath class='st95' d='M834.9,156.7H814V226h-20v-69.3h-21.9v-15.4h64.7L834.9,156.7z'/%3e%3cpath class='st95' d='M897.7,193.6c0,20.8-12,34.5-31.8,34.5c-19.7,0-31.8-12.7-31.8-34.6c0-20.8,12-34.4,31.8-34.4 C885.7,159,897.7,171.7,897.7,193.6z M854.2,193.5c0,14.4,3.9,20.4,11.7,20.4s11.7-6.2,11.7-20.3c0-14.4-3.9-20.4-11.7-20.4 S854.2,179.4,854.2,193.5z'/%3e%3cpath class='st95' d='M969.1,193.6c0,20.8-12,34.5-31.8,34.5c-19.7,0-31.8-12.7-31.8-34.6c0-20.8,12-34.4,31.8-34.4 C957.1,159,969.1,171.7,969.1,193.6z M925.6,193.5c0,14.4,3.9,20.4,11.7,20.4s11.7-6.2,11.7-20.3c0-14.4-3.9-20.4-11.7-20.4 S925.6,179.4,925.6,193.5z'/%3e%3cpath class='st95' d='M979.6,209.5v-74.2l19.3-2.1v75.5c0,2.7,1.1,4,3.4,4c1.2,0,2.3-0.2,3.2-0.6l3.7,13.7c-2.8,1.2-7,2.2-11.7,2.2 C986.2,228.1,979.6,221.3,979.6,209.5z'/%3e%3cpath class='st95' d='M1036.4,226h-19.3v-90.6l19.3-2.1V226z M1056.8,189.6L1078,226h-21.7l-19.3-35.4l19.8-29.4h19.7L1056.8,189.6 z'/%3e%3cpath class='st95' d='M1105.4,137.9c0,6.2-4.8,11-11.5,11s-11.4-4.8-11.4-11c0-6.2,4.6-11,11.4-11S1105.4,131.7,1105.4,137.9z M1103.7,226h-19.3v-64.9h19.3V226z'/%3e%3cpath class='st95' d='M1159.4,222.9c-4.9,3.4-11.7,5.1-17.6,5.1c-14.3-0.1-21.3-8.2-21.3-23.5v-30.1h-9.2v-13.4h9.2v-14l19.3-2.2 v16.2h14.9l-2.1,13.4h-12.8v29.8c0,6.2,2.1,8.4,6.1,8.4c2.2,0,4.4-0.6,7-2.1L1159.4,222.9z'/%3e%3c/g%3e%3c/svg%3e ");
```

Comment:
I don't like that we inline the SVG here. Instead, we should commit an SVG file and add code to generate this from the SVG file.

Reply:
We commit the SVG file inside of the CSS as the build output is a single HTML file. Was looking at WebPack plugins to do this such as https://github.com/webpack-contrib/svg-inline-loader but could not get them to work with version of WebPack we are using. Happy to file an improvement ticket for this to look at this at a later time.